.gitignore

```diff
@@ -3,3 +3,4 @@ vendor
 composer.phar
 composer.lock
 .phpunit.result.cache
+phpunit.xml
```

Comment:
you need to also ignore the `.phpunit.cache` folder (as PHPUnit 10 changed the location of the cache).

Note that you will also need to amend your commit to avoid committing the cache (which is currently part of the PR)